docs/gencertchain.md

```diff
@@ -1,19 +1,21 @@
-# gencertchain
+# gencertchain - A command-line tool for generating Certificate-Chain CBOR file
 
-A command-line tool for generating Certificate-Chain CBOR file.
-
-## Usage
+## SYNOPSIS
 
 This command generates an Certificate-Chain CBOR file.
 
 ```ShellSession
 $ gencertchain -certificate ./prime256v1.pem > certchain.cbor
 ```
 
-## Options
+## OPTIONS
 
 - `-help`:                 Show help message.
+
```

Comment:
Remove the dashes similar to `gensxg.md`?  This will look more similar to `ls`, etc. man pages.

Reply:
OK, removed.